otter_welcome_buddy/database/db_guild.py

```diff
@@ -21,8 +21,15 @@ def get_guild(
     def insert_guild(
         guild_model: GuildModel,
         session: Session,
+        override: bool = False,
     ) -> GuildModel:
         """Static method to insert a guild record"""
+        if not override:
+            current_guild_model = DbGuild.get_guild(guild_id=guild_model.id, session=session)
+            if current_guild_model is None:
```

Comment:
Why do we use `is not` and for the override, do we use the __bool__ method?

Reply:
Where do we use `is not`? I don't get the question